src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -172,8 +173,8 @@ private void runGcloudCommand(List<String> args, String... topLevelCommand)
   // class's main configured ProcessRunner should be used.
   private String runSynchronousGcloudCommand(List<String> args)
       throws ProcessRunnerException {
-    validateCloudSdk();
-
+    validateCloudSdkLocation();
```

Comment:
FYI I believe @joaoandremartins is working on some changes to validation in this library that uses an enum type for validation errors, building from [what we already do](https://github.com/GoogleCloudPlatform/google-cloud-intellij/blob/master/google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/sdk/DefaultCloudSdkService.java#L104-L132) in the intellij plugin.  When that's done it would remove the need to use unchecked exceptions.